libs/prelude/Prelude/Strings.idr

```diff
@@ -340,6 +340,20 @@ toUpper x with (strM x)
   strToLower (strCons c cs) | (StrCons c cs) =
     strCons (toUpper c) (toUpper (assert_smaller (strCons c cs) cs ))
 
+||| Convert a number string to int.
+|||
+||| ```idris example
+||| parseInt "123"
+||| ```
+parseInt : String -> Maybe Int                                                                                                                                                       
+parseInt s with (strM s)                                                                                                                                                             
+  parseInt ""             | StrNil         = Nothing                                                                                                                                 
+  parseInt (strCons x xs) | (StrCons x xs) = parseIntAux (unpack xs) (ord x - 48) where                                                                                              
```

Comment:
Yeah, sorry I meant to change that but forgot while doing the indentation check :tongue: